pages/index.tsx

```diff
@@ -386,8 +386,15 @@ export const getStaticProps: GetStaticProps<HomeProps> = async () => {
     }
   });
 
+  const thirdPartyEntryPool = [];
+  for (let i = 0; i < POOL_NUM_THIRD_PARTY; i++) {
+    const s = Math.floor(thirdPartyEntries.length * Math.random());
+    thirdPartyEntryPool.push(thirdPartyEntries[s]);
+    thirdPartyEntries.splice(s, 1);
+  }
```

Comment:
This whole random selection along with the one in the useEffect should be moved to a utility method. Where should I put it? Is putting it to `manual_utils` or `register_utils` acceptable?

Reply:
I think in `pages/index.ts` is probably fine as that is the only place its used :-)